index.html

```diff
@@ -3,6 +3,12 @@
   <head>
     <meta charset="UTF-8" />
     <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
```

Comment:
try to import fonts from css files, not from index.html. styles related imports should in in styles files.